tools/test.py

```diff
@@ -957,8 +957,14 @@ def GetVm(self, arch, mode):
 
   def GetTimeout(self, mode, section=''):
     timeout = self.timeout * TIMEOUT_SCALEFACTOR[ARCH_GUESS or 'ia32'][mode]
-    if section == 'pummel' or section == 'benchmark' or section == 'wpt':
+    if section == 'pummel' or section == 'benchmark':
       timeout = timeout * 6
+    # We run all WPT from one subset in the same process using vm contexts.
+    # As the number of the tests grow, it can take longer to run some of the
+    # subsets, but it's still overall faster than running them in different
+    # processes or different workers.
```

Comment:
I think we use workers now.

Reply:
Oh, yeah, my brain was still living in the era where we ran the tests with vm contexts, now it makes even more sense why the tests tend to timeout.... Updated, thanks.